tests/test_admin.py

```diff
@@ -131,3 +131,60 @@ def test_list_url_is_not_trucated_under_200_chars(admin_client):
     truncated_url = doc.find(".field-truncated_url").text()
 
     assert truncated_url == "/a1b2c3d4e/some-path"
+
+
+@pytest.mark.django_db
+def test_response_content_length_empty(admin_client):
+    """Assert that the length of the content of the response is empty"""
+
+    log = OutgoingRequestsLog.objects.create(
+        id=1,
+        req_body=b"",
+        res_body=b"",
+        timestamp=timezone.now(),
+    )
+    url = reverse(
+        "admin:log_outgoing_requests_outgoingrequestslog_change", args=(log.pk,)
+    )
+
+    response = admin_client.get(url)
+    assert response.status_code == 200
+
+    html = response.content.decode("utf-8")
+    doc = PyQuery(html)
+    request_body = doc.find(".field-request_body .readonly").text()
+    response_body = doc.find(".field-response_body .readonly").text()
+    content_length = doc.find(".field-response_content_length .readonly").text()
+
+    assert request_body == "-"
+    assert response_body == "-"
+    assert content_length == "0"
+
+
+@pytest.mark.django_db
+def test_response_content_length_displayed(admin_client):
+    """Assert the length of the content of the response is displayed"""
+
+    log = OutgoingRequestsLog.objects.create(
+        id=1,
+        req_body=b"Test request",
+        res_body=b"Test Response",
+        timestamp=timezone.now(),
+    )
+    url = reverse(
+        "admin:log_outgoing_requests_outgoingrequestslog_change", args=(log.pk,)
+    )
+
+    response = admin_client.get(url)
+    assert response.status_code == 200
+
+    html = response.content.decode("utf-8")
+    doc = PyQuery(html)
+    request_body = doc.find(".field-request_body .readonly").text()
+    response_body = doc.find(".field-response_body .readonly").text()
+    content_length = doc.find(".field-response_content_length .readonly").text()
+
+    assert request_body == "Test request"
+    assert response_body == "Test Response"
+    assert content_length == "13"
+    assert content_length == log.response_content_length
```

Comment:
Since #44 specifically mentions being able to see the content length in the listview, could you also do a request to the changelist to verify that the content length is properly displayed there as well?